packages/form-builder/addon/components/cfb-form-list/copy-modal.hbs

```diff
@@ -0,0 +1,49 @@
+{{#if this.visible}}
+  <UkModal
+    @visible={{this.visible}}
```

Comment:
This will probably give you some bad UX. I understand the reason for it, but if we just moved the button to the detail view, we wouldn't need to think about it..

Reply:
Not addressed yet

Reply:
I've already moved the `{{#if this.visible}}` into the modal body around the `ValidatedForm`. Fixes the *bad UX* as the modal is animating properly and gives enough feedback for the tests. So I guess it's *OK* now?!